consensus/ibft/snapshot_test.go

```diff
@@ -711,11 +710,6 @@ func TestSnapshot_ProcessHeaders(t *testing.T) {
 						})
 					}
 					if !resSnap.Equal(snap) {
-						fmt.Println("-- wrong result --")
-						fmt.Println(resSnap.Set)
-						fmt.Println(snap.Set)
-						fmt.Println(resSnap.Votes)
-						fmt.Println(snap.Votes)
 						t.Fatal("bad")
```

Comment:
I think this is out of the scope for this PR, since we have this horrible test error handling logic in a lot of places:
![Uploading Screenshot 2021-12-27 at 16.04.37.png…]()

Reply:
Don't you feel a little bit sad to leave it like this? :)

Reply:
Please see the screenshot and tell me if I need to feel sad 😞